debian/control

```diff
@@ -6,11 +6,9 @@ Build-Depends: cmake (>= 2.8.11),
                debhelper (>= 9),
                dh-systemd,
                dpkg-dev (>= 1.17),
-               gcc,
                libnl-3-dev,
                libnl-route-3-dev,
                libudev-dev,
-               make,
                ninja-build,
```

Comment:
Is that review system publicly available?

Assuming that I correctly read the ill-formatted pasted commit (please use block format the next time), Talat's commit is identical to my commit [debian: Correct name of ninja-build dependency](https://github.com/linux-rdma/rdma-core/pull/54/commits/0c142bcd127901faff3b27d6c4a94452567c4c4f) which is the second commit of this merge proposal.